### ISSUE TITLE
Patch for /trunk/cpplint/cpplint.py

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google/styleguide#16]
```
Adding streambuf as a valid CPP header. You can see that this include name is 
valid:

http://www.cplusplus.com/reference/streambuf/

cpplint.py was giving a false positive with the valid code:

#include <streambuf>

But was passing with the invalid code (didn't compile):

#include <streambuf.h>
```

Original issue reported on code.google.com by `chris.da...@gmail.com` on 17 Jun 2013 at 3:14

Attachments:
- [cpplint.py.patch](https://storage.googleapis.com/google-code-attachments/google-styleguide/issue-16/comment-0/cpplint.py.patch)
